examples/gameroom/daemon/src/authorization_handler/mod.rs

```diff
@@ -81,8 +81,21 @@ pub fn run(
         .map(|gameroom| resubscribe(&splinterd_url, gameroom, &db_conn))
         .try_for_each(|ws| igniter.start_ws(&ws))?;
 
+    let since_param = helpers::get_last_updated_proposal_time(&pool)?
+        .map(|time| {
+            format!("?since={}", {
+                time.duration_since(SystemTime::UNIX_EPOCH)
+                    .map(|duration| duration.as_millis())
+                    .unwrap_or(0)
+            })
+        })
+        .unwrap_or_else(|| String::from(""));
```

Comment:
Maybe the parameter should simply be left off when the auth handler doesn't have a last updated time, rather than providing an empty string (which isn't an integer as the API spec requires) for the param?

Reply:
That is how the parameter is being left off. That produces either:
```
{
    "?since={some number}"
} else {
    ""
}
```

Reply:
It could return the full path in either block, I suppose.

Reply:
Oops, misread this. Thanks for clarifying.